components/post_view/post_body_additional_content/post_body_additional_content.jsx

```diff
@@ -152,13 +152,13 @@ export default class PostBodyAdditionalContent extends React.PureComponent {
 
         if (embed) {
             const toggleable = this.isEmbedToggleable(embed);
-            const prependToggle = (/^\s*https?:\/\/.*$/).test(this.props.post.message);
 
             return (
                 <div>
-                    {(toggleable && prependToggle) && this.renderToggle(true)}
-                    {this.props.children}
-                    {(toggleable && !prependToggle) && this.renderToggle(false)}
+                    <div className='file-preview__link'>
+                        {this.props.children}
+                        {toggleable && this.renderToggle(true)}
```

Comment:
Render toggle does not longer receive any value, so we can call it just as `renderToggle()`